buildSrc/src/main/kotlin/kotlinx.fuzz.example-module.gradle.kts

```diff
@@ -11,6 +11,7 @@ version = VERSION
 
 repositories {
     mavenCentral()
+    maven(url = "https://plan-maven.apal-research.com")
```

Comment:
Why do we need to add our maven repo here? Are we switching all tests to use our version of Jazzer?

Reply:
I thought so, because, for example, gradle module uses jazzer as a dependency and therefore we need to have this line in `build.gradle.kts` of `kotlinx.fuzz.gradle` as well as in `kotlinx.fuzz.jazzer`. We can just use it in both places, but then we will need it in all alternatives for gradle. It still makes sense to remove it from buildSrc, because not every module uses it, but still some of them do

Reply:
But this is the example module, which is only used in `kotlinx.fuzz.example`, i.e. it is necessary only for our old implementations. In does not affect `kotlinx.fuzz.gradle` and `kotlinx.fuzz.jazzer`